### PR TITLE
Added option to ignore safety checks

### PR DESCRIPTION
**Pull request by Physarah**
This is the fix for #20 . It adds the option to read in and ignore values from config. If it is False, no weather params are ignored, but if there are 'rain', 'clouds', 'gust' or 'wind' specified in a list they are then ignored when making the safety decision.

### CODE REVIEW

**Comment by Physarah** [comment]
@danjampro @wtgee could you take a look at this for me?

File: aag/weather.py
Comment (wtgee) [inline aag/weather.py:751]: It's a bit better to set this to `None` rather than, which indicates we don't have a default but is (somewhat) consistent with the same data type (list of strings) as opposed to `False`.

```suggestion
    def make_safety_decision(self, current_values, ignore=None):
```

File: aag/weather.py
Comment (wtgee) [inline aag/weather.py:755]: American english please :)

```suggestion
        ignore: list of safety params to ignore. Can be 'rain', 'wind', 'gust' or 'cloud'. If None (default) nothing is ignored.
```

File: aag/weather.py
Comment (wtgee) [inline aag/weather.py:780]: Assuming the `None` above, you can listify it with our other tools. Also need to remove the `else` statement as you always want to assign to `safe`.  I'm also logging the ignored value, so pop it rather than delete it so we have access to it after. If this will be too noisy could leave off.

```suggestion
        from panoptes.utils.utils import listify  # at top of file
        
        if ignore is not None:
            for weather_to_ignore in listify(ignore):
                ignored_value = safety_params.pop(weather_to_ignore)

                # Warn if ignoring an unsafe value.
                if ignored_value is False:
                    logger.warning(f'Ignored unsafe value: {weather_to_ignore}={ignored_value}')
                
        # Do final safety check.
        safe = all(safety_params.values())
```

**Review by wtgee — APPROVED** [review]
A few minor suggestions. One improvement might be to read the ignored params from the config file, but could be left for a separate PR.

Thanks for getting this done!

**Comment by Physarah** [comment]
Ok, looks like this is good to go now all comments are addressed. Grabbing ignored params from the config file will come in the next PR :) Thanks all!

**Comment by Physarah** [comment]
@danjampro , @wtgee I can't merge this as I don't have write access, so go ahead when you're ready

**Comment by Physarah** [comment]
Added getting value from config with None default. Not sure if this is the correct POCS way to do this @wtgee @danjampro ?

File: config.yaml
Comment (wtgee) [inline config.yaml:13]: `ignore` is a little bit generic, maybe `ignore_unsafe`?

```suggestion
    ignore_unsafe: None # can be a list, e.g. 'rain','cloud','gust','wind'
```

**Review by wtgee — APPROVED** [review]
👍🏼 Thanks for the changes!

**Pull request merged**
